introduction.tex

```diff
@@ -1,15 +1,31 @@
 \section{Introduction} \label{sec:intro}
 
-The current situation is  that, only the Science Pipelines is released as follows:
+This document collects the needs in therms of releses from different stakehoders, section \ref{sec:reqs}.
+These needs are presented as requirements, that shall not be confused with project requirements, like for example the \gls{DMS} requirement specification, documented in \citeds{LSE-61}.
+Section \ref{sec:process} provides an overview of the release process.
+
+Appedix \ref{sec:defs} provides definitions of the main objects involved in the release activities. 
+Those definitions are given in order to provide a common understanding.
+In the same line, appendix \ref{sec:swptree} gives an overview of the software products part of the \gls{DM}, that are involved in this process.
```

Comment:
appendix \ref{sec:swptree} also an overview of the software products part of the \gls{DM} system that are involved in this process.

Reply:
I suppose the verb needs to be kept...

Reply:
Yes- I just updated the original comment